microcosm_flask/conventions/crud.py

```diff
@@ -44,15 +45,22 @@ def register_search_endpoint(graph, obj, path_prefix, func, request_schema, resp
     :param response_schema: a marshmallow schema to encode (a single) response item
     """
 
+    paginated_list_schema = make_paginated_list_schema(obj, response_schema)()
```

Comment:
I'm using marshmallow schemas to generate the JSON schema definitions in the swagger responses. As a result, I needed to create a paginated list schema for every generated search endpoint -- I'd avoided this so far.

Note that I haven't fully integrated this schema; the response is still coming from the `PaginatedList.to_dict()` method.